integrations/gtag/lib/index.js

```diff
@@ -19,39 +19,33 @@ var extend = require('extend');
 
 var GTAG = (module.exports = integration('Gtag')
   .global('gtagDataLayer')
-  .option('gaWebMeasurementId', '')
-  .option('gaWebAppMeasurementId', '')
   .option('awConversionId', '')
   .option('dcFloodLightId', '')
-  .option('trackAllPages', false)
   .option('trackNamedPages', true)
   .option('trackCategorizedPages', true)
-  .option('gaOptions', {
-    classic: false,
-    enhancedEcommerce: false,
-    setAllMappedProps: true,
-    anonymizeIp: false,
-    domain: 'auto',
-    enhancedLinkAttribution: false,
-    optimize: '',
-    sampleRate: 100,
-    siteSpeedSampleRate: 1,
-    sendUserId: false,
-    useGoogleAmpClientId: false
-  })
-  .option('includeSearch', false)
-  .option('anonymizeIp', false)
-  .option('domain', 'auto')
+  .option('includeQueryString', false)
+  .option('gaWebMeasurementId', '')
+  .option('gaWebAppMeasurementId', '')
+  .option('gaCustomDimensions', {})
+  .option('gaCustomMetrics', {})
+  .option('gaContentGroupings', {})
+  .option('gaEnhancedEcommerce', false)
+  .option('gaAnonymizeIp', false)
+  .option('gaCookieDomain', 'auto')
+  .option('gaEnhancedLinkAttribution', false)
+  .option('gaOptimizeContainerId', '')
+  .option('gaSampleRate', 100)
+  .option('gaSendUserId', false)
+  .option('gaUseAmpClientId', false)
+  .option('gaSiteSpeedSampleRate', 1)
+  .option('gaSetAllMappedProps', false)
```

Comment:
Nested objects aren't supported setting types in Partner Portal so all GA options needed to be moved to the top level.